docs/supported_models/transformers_fallback.md

```diff
@@ -0,0 +1,58 @@
+# Transformers fallback in SGLang
+
+`sglang` can fallback to using models that are available in `transformers`. This works for most decoder-style language models and support for vision-language models is coming soon!
+
+## Example launch Command
+
+By default, we will use sglang implementation if it is available. Otherwise, we will fallback to transformers one. However, you can switch the implementation by setting `impl` to `transformers`.
```

Comment:
```suggestion
By default, we will use sglang implementation if it is available. Otherwise, we will fall back to transformers one. However, you can switch the implementation by setting `impl` to `transformers`.
```